src/js/experimental/media-chrome-listbox.js

```diff
@@ -23,11 +23,11 @@ template.innerHTML = `
   }
 
   ::slotted(media-chrome-listitem[aria-selected="true"]) {
-    background-color: var(--media-listbox-selected-background, rgba(122,122,184, .8));
+    background: var(--media-listbox-selected-background, rgb(122 122 184 / 0.7));
```

Comment:
- Changing to apply to `background` instead of `background-color` for consistency
- Changing default transparency/alpha to `0.7` for consistency
- Should consider renaming variable to have a more consistent naming convention.

Reply:
I don't think it's going to be a problem for us, but I'm always wary of changes of `background-color` to `background` because it could reset other props that `background` is a shorthand for as well.